### ISSUE TITLE
Use dot based map

### ISSUE THREAD

**Issue by gregdyke** [ManSangSin/Rhythm-Code#18]
## Technical task

We can settle on one dot based map and each rhythm/rhythm code can be associated with one specific dot (or one specific dot coordinate as is convenient for the implementer) on the map

- **As a** .............

- **I Want** .............

- **So That** .............

## Acceptance Criteria

1. Given .............
   - When .............
   - Then .............
2. Given .............
   - When .............
   - Then .............

## Implementation Notes

1.
2.

## Checklist:

- [ ] I have defined the user story
- [ ] I have defined some acceptance criteria
- [ ] I have included any implementation notes
- [ ] I have included all relevant links

**Comment by gregdyke** [comment]
This one https://upload.wikimedia.org/wikipedia/commons/a/a0/World_map_with_points.svg or the one from the .ai file attached in slack could be used

**Comment by mferryRV** [comment]
Does this replace #5 in that this should be our only map? If so, it is an easy fix.

**Comment by gregdyke** [comment]
That's correct. As mentioned here on Slack https://codeyourfuture.slack.com/archives/C05SUQ3K5DY/p1696065119931789

**Comment by mferryRV** [comment]
@ManSangSin - As part of this ticket, I would ask that you make sure the icons appear in the right place.

In sprint planning we discussed PX CSS positioning or selecting SVG path elements as possible solutions for locating the dots.

Your decision was to select SVG path IDs and go from there.

**Comment by mferryRV** [comment]
Need to have this thought through and communicated by Tuesday.
